fox_scraper/fox_scraper.py

```diff
@@ -14,97 +14,75 @@
         1. The scraper input - a site address link
         2. The scraper output - all Fox’s catalog
         3. The scraper will use two main functions:
-            1. scrap_fox_runner() - this will be the main function for the site scraping it will retrieve
+            1. runner() - this will be the main function for the site scraping it will retrieve
                the main categories (women, men, kids, baby) and use helper functions for retrieving other categories and cloths
             2. publish_data_to_rabbitmq() - this function will publish all data towards rabbitmq
 '''
 
 
 from bs4 import BeautifulSoup
+from fox_helper import FoxItem
+from fox_helper import FoxSender
 import requests
 
-class fox_item(object):
-    def __init__(self, item_img_id, item_main_category, item_type, item_name, item_price):
-        self.item_img_id = item_img_id
-        self.item_main_category = item_main_category
-        self.item_type = item_type
-        self.item_name = item_name
-        self.item_price = item_price
-
-    def print_item(self):
-        print "item image id:", self.item_img_id
-        print "item main category:", self.item_main_category
-        print "item type:", self.item_type
-        print "item name:", self.item_name
-        print "item price: %.2f " %self.item_price
-
 
 class Scraper:
```

Comment:
It would be nice if you add here a docstring with the explanation of how to use this class